pkg/envoy/xds/server.go

```diff
@@ -168,7 +172,7 @@ type perTypeStreamState struct {
 
 	// nonce is the nonce sent in the last response to a request for this
 	// resource type.
-	nonce string
+	nonce uint64
```

Comment:
That field is updated, but never read anymore. Just remove it and the code that updates it.

Reply:
Done.